qiskit/visualization/state_visualization.py

```diff
@@ -330,19 +434,138 @@ def plot_state_city(rho, title="", figsize=None, color=None,
            plot_state_city(job.get_statevector(qc), color=['midnightblue', 'midnightblue'],
                 title="New State City")
     """
-    if not HAS_MATPLOTLIB:
-        raise ImportError('Must have Matplotlib installed.')
-    rho = _validate_input_state(rho)
-
     num = int(np.log2(len(rho)))
-    # get the real and imag parts of rho
-    datareal = np.real(rho)
-    dataimag = np.imag(rho)
 
     # get the labels
     column_names = [bin(i)[2:].zfill(num) for i in range(2**num)]
     row_names = [bin(i)[2:].zfill(num) for i in range(2**num)]
 
+    fig = _plot_matrix_city(mat=rho, row_names=row_names, column_names=column_names,
+                            oper_name='$\\rho$', title=title, figsize=figsize,
+                            color=color, alpha=alpha, ax_real=ax_real, ax_imag=ax_imag)
+
+    if ax_real is None and ax_imag is None:
+        if get_backend() in ['module://ipykernel.pylab.backend_inline',
+                             'nbAgg']:
+            plt.close(fig)
+        return fig
+
+
+def plot_quantum_channel_city(channel, title="", figsize=None, color=None,
+                              alpha=1, ax_real=None, ax_imag=None, basis=None):
+    """City plot of the Chi-matrix representation of a quantum channel.
+
+    Plot two 3d bar graphs (two dimensional) of the real and imaginary
+    part of the quantum channel.
+
+    Args:
+        channel (QuantumChannel): Arbitrary QuantumChannel object to visualize.
```

Comment:
This should say it takes a QuantumCircuit or channel since that it now what it does.